SU-HSDO/stanford_mrc
# rm stanford basic

# READY FOR REVIEW

# Summary
- Removed older version of Decanter
- Added new subtree (master branch)

# Needed By (Date)
- As soon as possible (not needed before close of current sprint)

# Urgency
- N/A

# Steps to Test

1. Check out this branch
2. Check out the new branch in mrc_blt
3. Run composer update
4. Verify that the theme is no longer living in the profiles theme folder and is now living in Drupal (docroot) theme folder

# Affected Projects or Products
- stanford_mrc
- MRC

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
